Projects/App/Sources/Global/Extension/UITableView+.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  UITableView+.swift
+//  App
+//
+//  Created by Chanhee Jeong on 2022/10/25.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import DesignSystem
+
+extension UITableView {
+    
+    enum EmptyViewType {
+        case search
+        case noresult
+    }
+    
+    func setEmptyView(message: String, type: EmptyViewType) {
+        
+        let emptyView = UIView(frame: CGRect(x: self.center.x,
+                                             y: self.center.y,
+                                             width: self.bounds.size.width,
+                                             height: self.bounds.size.height))
+           
+        let iconView: UIImageView = {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            $0.image = UIImage((type == .search) ?  .img_emptyfriends_search : .img_emptyfriends_noresult)
+            return $0
+        }(UIImageView())
```

Comment:
p1;

지금 아이콘에 그림자가 안 들어가있네요. applyFigmaShadow 메서드 만들어놓으셨으니까 적용해주시죠!

```suggestion
        let iconView: UIImageView = {
            $0.translatesAutoresizingMaskIntoConstraints = false
            $0.image = UIImage((type == .search) ?  .img_emptyfriends_search : .img_emptyfriends_noresult)
            $0.layer.applyFigmaShadow(
                color: .black,
                opacity: 0.1,
                xCoord: 0,
                yCoord: 0,
                blur: 5,
                spread: 0
            )
            return $0
        }(UIImageView())
```

Reply:
새벽 5시엔,, 이런일이 벌어지는군요 반영이요

Reply:
<img width="230" alt="스크린샷 2022-10-25 오후 11 04 23" src="https://user-images.githubusercontent.com/63157395/197795101-db531422-7325-4368-9c17-c3d549a97ce1.png">